spec/queries/users_by_company_query_spec.rb

```diff
@@ -18,17 +20,23 @@
 
   subject { UsersByCompanyQuery.new(company) }
 
-  describe '#not_allocated_including' do
-    context 'when passes user allocated' do
-      it 'return all users not allocated including user passed' do
-        result = subject.not_allocated_including(user_allocated)
+  describe '#active_engineers' do
+    context 'when users are engineers and actives' do
+      it 'return all users' do
+        result = subject.active_engineers
         expect(result).to containing_exactly(user_not_allocated, user_allocated)
```

Comment:
not your code, but if you could change this to `expect(result).to contain_exactly` that'd be great ☕